mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -1316,9 +1316,14 @@ class LogisticRegressionModel private[spark] (
   }
 }
 
-
 @Since("1.6.0")
 object LogisticRegressionModel extends MLReadable[LogisticRegressionModel] {
+  case class Data(
```

Comment:
I don't know why it fails `MLSuite`, but we should not make it public.
It seems `private[spark]` can work. https://github.com/apache/spark/pull/50760